ase/src/main/java/com/ase/ase/rest/controller/BasicCovidController.java

```diff
@@ -38,7 +36,23 @@
                 new HospitalSituationPerDateDto(provinceId, Arrays.asList(
                         new HospitalSituationPerDate("10.10.2020", 12, 10),
                         new HospitalSituationPerDate("12.10.2020", 12, 12))));
+
     }
 
+    @CrossOrigin
+    @GetMapping(value = "/tableData/{province-id}", produces = "application/json")
```

Comment:
Do not call the rout "tableData". The rest service should not know that the data is user for a table. It could also be used for other things like the map for example. Then I don't want to call .../tableData. Rename it in something it really stands for.

Reply:
maybe you can call it something like summary oder generalization